docs/modules/develop/pages/classes/events.adoc

```diff
@@ -0,0 +1,23 @@
+= Events
+
+The concept regarding the notitication is being described in the xref:develop:notifications.adoc[Notifications] page.
+
+Event classes are located in the `app/events/decidim/<my_plugin>` directory, and named: `<my_event>_event.rb`.
+
+```ruby
+
```

Comment:
According to the line 5, this should be: 
```suggestion
# frozen_string_literal: true

# app/events/decidim/my_plugin/my_event_event.rb
```

That sounds really bad, but we also have the my_cell_cell.rb example